src/Task/Occurrence.ts

```diff
@@ -88,7 +88,7 @@ export class Occurrence {
      *
      * @param nextReferenceDate
      */
-    public next(nextReferenceDate: Date): Occurrence {
+    public next(nextReferenceDate: Date, dropScheduledDate: boolean = false): Occurrence {
```

Comment:
Please update the JSdocs for the new parameter...

Reply:
Done.